api/integrations/launch_darkly/tasks.py

```diff
@@ -1,12 +1,9 @@
-from django.db import transaction
-
 from integrations.launch_darkly.models import LaunchDarklyImportRequest
 from integrations.launch_darkly.services import process_import_request
 from task_processor.decorators import register_task_handler
 
 
 @register_task_handler()
 def process_launch_darkly_import_request(import_request_id: int) -> None:
-    with transaction.atomic():
-        import_request = LaunchDarklyImportRequest.objects.get(id=import_request_id)
-        process_import_request(import_request)
+    import_request = LaunchDarklyImportRequest.objects.get(id=import_request_id)
+    process_import_request(import_request)
```

Comment:
Just so I'm clear here: by removing the `transaction.atomic` context manager, we're allowing imports to partially complete, right? So we might end up with half the data from LD imported? What's the thought proces here?

Reply:
We're solving a couple of problems here:

1. Created entities not available to the audit log workers ([related sentry issue](https://flagsmith.sentry.io/issues/4555706486/?alert_rule_id=14542360&alert_type=issue&notification_uuid=22537a24-c37e-433f-b1b2-bbc678327d51&project=5544478&referrer=slack))
3. Potentially long transactions

We'll also be able to surface detailed errors in import request statuses with attribution to specific entities that caused them, should we encounter such errors in the future (at this moment everything is expected to work smoothly).

The basic UX motivation/assumption for this is that having a partial import is better than having nothing imported at all. If we rethink this, we should solve 1. with delaying audit log task processor calls until the transaction ends.

Reply:
>The basic UX motivation/assumption for this is that having a partial import is better than having nothing imported at all.

I would probably argue the opposite here to be honest, but I don't think it's a massive issue.

Reply:
Okay. Feel free to unresolve.